fixtures/tests/merge_fastqs/10151_F_13.file.json

```diff
@@ -0,0 +1,54 @@
+[
```

Comment:
Currently, the test only checks for when merging is required. I made a PR to export File and File Metadata fixtures from Django Admin, when that gets merged in I'll add cases for other samples in this request.

Reply:
👍